package-lock.json

```diff
@@ -4418,6 +4418,12 @@
         "ini": "^1.3.4"
       }
     },
+    "global-registry-nodejs-client": {
+      "version": "github:CruGlobal/global-registry-nodejs-client#75beee7a2e7a63d1d1bac5218b033bae50596800",
```

Comment:
@pkoz I think it happened again. This commit isn't in that repo anymore. Maybe Travis didn't complain, because it was in that repo at the time of the build?

Anyway, can you make another small PR to fix this?